phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexScrutinyMapper.java

```diff
@@ -227,9 +236,14 @@ private void processBatch(Context context)
             Map<String, Pair<Long, List<Object>>> targetPkToSourceValues =
                     buildTargetStatement(targetStatement);
 
+            preQueryTargetTable();
```

Comment:
Nit: This fn implementation has nothing specific to target table. Also, the clock change applies to source table too. Is there any better name? e.g. preBatchScrutiny(), preScrutinyQueries()?

Reply:
this clock change has to happen before the target table is queried and hence the name